frontend/src/App.js

```diff
@@ -34,22 +59,49 @@ const App = () => {
         },
       });
     } catch (error) {
-      console.error("Could not perform logout of user, wiping out local session.", error);
+      console.error(
+        "Could not perform logout of user, wiping out local session.",
+        error
+      );
     }
     deleteBackendAuthToken();
     setAuthorized(false);
-  }
+  };
 
   return (
-    <div className="App">
-      {
-        authorized ? 
-          <Dashboard user={{name: authorized}} logoutAction={logout}/> 
-        : 
-          <LoginAndSignupForm loginAction={login}/>
-      }
+    <ThemeProvider theme={theme}>
+    <div className="App">      
+      {authorized ? (
+        <Dashboard user={{ name: authorized }} onLogoutHook={logout} />
+      ) : (
+        <React.Fragment>
+          <img
+            style={{
+              backgroundImage: `url("peas.jpg")`,
+              backgroundSize: "cover",
```

Comment:
Styles for this could be used inside **styles/theme.js**, maybe we should move them there?

Reply:
OK